app/services/deal_analytic_service.rb

```diff
@@ -424,6 +429,140 @@ def check_deal_analytics(merchant_id)
       deal_analytics
     end
 
+    # returns conversion rate of num wishlisters to viewers in percentage
+    def get_wishlist_to_views(deal_id)
+      push_date = Deal.find(deal_id).push_date
+      num_wishlist = WishService.num_wishlist_deal(deal_id, push_date).to_f
+      user_id = WishService.get_user_id(deal_id, push_date)
+      num_views = Viewcount.where(user_id: user_id, deal_id: deal_id).uniq.where(:entry => 'merchant_push_notification').count.to_f
```

Comment:
@Woonyong92 Have u confirmed w @junwen29 if there is an entry point for "merchant push notification"?